identity/client/e2e/pages/GroupsPage.ts

```diff
@@ -33,18 +33,19 @@ export class GroupsPage {
   readonly closeDeleteGroupModal: Locator;
   readonly deleteGroupModalCancelButton: Locator;
   readonly deleteGroupModalDeleteButton: Locator;
+  readonly emptyState: Locator;
 
   constructor(page: Page) {
     this.page = page;
     // List page
     this.groupsList = page.getByRole("table");
-    this.createGroupButton = page.getByRole("button", {
-      name: /create group/i,
-    });
+    this.createGroupButton = page
+      .getByRole("button", {
+        name: /create( a)? group/i,
+      })
+      .first();
```

Comment:
Why do we need the `first()` here? Should we make it configurable if there's more than 1 item per page?

Reply:
I don't think we need it actually, I've removed it.